web-common/src/components/forms/Input.svelte

```diff
@@ -33,6 +33,7 @@
   export let fields: string[] | undefined = [];
   export let disabled = false;
   export let disabledMessage = "No valid options";
+  export let readonly = false;
```

Comment:
Are there any `<Input />` instances using this prop? I'm not seeing it used at first glance.

Reply:
previously added variable name to be readonly, removing it now